### DISCUSSION TITLE
能自訂邊緣滾動的偵測範圍嗎

### DISCUSSION THREAD

**Discussion by gcobc12632** [shuax/MouseInc.Settings#218] [Ideas]
我目前的設定是 將滑鼠移動到螢幕上方邊緣時 滾動滑鼠切換瀏覽器的標籤頁

但一定要在邊緣才能生效 不知道有沒有辦法自訂範圍？

例如 只要滑鼠移動到螢幕上方5%內 或是50px內 都算在邊緣

**Comment by shuax** [comment]
用单独的浏览器增强软件吧，就不用在屏幕边缘了。

**Reply by gcobc12632** [reply]
有推薦的瀏覽器增強嗎？對此不太熟悉

其實我之前一直都在使用你做的FireDoge 已經用好幾年了

但前幾天Firefox更新到96.0後 FireDoge就失效了 所以才來用MouseInc替代

我主要是想還原「滑鼠停在標籤頁的時候捲動滾輪 就能切換瀏覽器的標籤頁」這個功能

雖然邊緣滾動大致上可以符合需求 但偵測範圍不像以往的那樣寬鬆 所以才來這裡問有沒有解法
